flink-end-to-end-tests/test-scripts/common_mesos_docker.sh

```diff
@@ -35,8 +35,8 @@ start_time=$(date +%s)
 # make sure we stop our cluster at the end
 function cluster_shutdown {
   docker-compose -f $END_TO_END_DIR/test-scripts/docker-mesos-cluster/docker-compose.yml down
-  rm -rf ${INPUT_VOLUME}/log
-  rm -rf ${INPUT_VOLUME}/tmp
+  sudo rm -rf ${INPUT_VOLUME}/log
```

Comment:
This is not a comment. Just wondering why is sudo privilege needed to remove these files?

Reply:
I could not reproduce the issue locally, so it hard to figure out the root cause. I guess something went wrong when the docker started and mounted the directory.

Reply:
So seems `sudo` just hides the problem that the files cannot be deleted with normal privilege.
I think it would be better to figure out the root cause.

Reply:
Agreed.
Yes, it just seldom repro in travis.

Reply:
I think the root cause of this case failure is that the files cannot be be deleted with normal privilege.
So I think this change is not a valid solution to fix this issue.

Reply:
Yes. I have edited the PR description. Please take a look :) .